packages/exoflex/example/webpack.config.js

```diff
@@ -1,79 +0,0 @@
-/* eslint-disable @typescript-eslint/no-var-requires */
```

Comment:
Just making sure here since we deleted the webpack config file from the example.

Does changes to exoflex sources reflect in the example project when running?

Reply:
Hmm I forgot about why we remove this. 
Maybe @ikusa, did you remember why we remove this? Iirc we do this together when discussing about creating a new expo example instead of upgrading it.

But to answer your question, yes changes on exoflex sources reflect directly on the running project.

Reply:
All good then.

For the backstory, older Expo web can't do that out of the box. So we had to customize webpack config so exoflex devs can see changes in the web example in real time.

If we don't need to do that anymore, thats great!